sway/commands/move.c

```diff
@@ -916,8 +907,7 @@ static struct cmd_results *cmd_move_to_position(int argc, char **argv) {
 	case MOVEMENT_UNIT_DEFAULT:
 		break;
 	case MOVEMENT_UNIT_INVALID:
-		sway_assert(false, "invalid x unit");
-		break;
```

Comment:
It seems like this was unreachable with the previous code, because there was a `lx.unit == MOVEMENT_UNIT_INVALID` check above? Why change it?